spec/models/miq_dialog_spec.rb

```diff
@@ -0,0 +1,28 @@
+describe MiqDialog do
+  context '.seed' do
+    it 'seeds from the core with correct metadata' do
+      root = Rails.root.join('product', 'dialogs', 'miq_dialogs')
+      allow(described_class).to receive(:find_by)
```

Comment:
here it is still, I need to stub `find_by` with `allow(described_class).to receive(:find_by)`  otherwise I get this:
same is true for `:sync_from_dir` below.


```
Failures:

  1) MiqDialog.seed seeds from the core with correct metadata
     Failure/Error: rec = find_by(:name => item[:name], :filename => item[:filename])

       #<MiqDialog(id: integer, name: string, description: string, dialog_type: string, content: text, default: boolean, filename: string, file_mtime: datetime, created_at: datetime, updated_at: datetime, href_slug: string, region_number: integer, region_description: string) (class)> received :find_by with unexpected arguments
         expected: ({:name=>"miq_host_provision_dialogs", :filename=>"miq_host_provision_dialogs.yaml"})
              got: ({:name=>"miq_provision_amazon_dialogs_template", :filename=>"miq_provision_amazon_dialogs_template.yaml"})
       Diff:
       @@ -1,3 +1,3 @@
       -[{:name=>"miq_host_provision_dialogs",
       -  :filename=>"miq_host_provision_dialogs.yaml"}]
       +[{:name=>"miq_provision_amazon_dialogs_template",
       +  :filename=>"miq_provision_amazon_dialogs_template.yaml"}]

     # ./app/models/miq_dialog.rb:36:in `sync_from_file'
     # ./app/models/miq_dialog.rb:20:in `block in sync_from_dir'
     # ./app/models/miq_dialog.rb:20:in `each'
     # ./app/models/miq_dialog.rb:20:in `sync_from_dir'
     # ./app/models/miq_dialog.rb:15:in `seed'
     # ./spec/models/miq_dialog_spec.rb:11:in `block (3 levels) in <top (required)>'
     # -e:1:in `<main>'
```